.github/workflows/lint-build-test.yml

```diff
@@ -15,18 +15,25 @@ jobs:
         uses: actions/checkout@v2
       # Install NPM dependencies, cache them correctly
       - name: Install dependencies
-        uses: cypress-io/github-action@v2
+        uses: cypress-io/github-action@v4
```

Comment:
Shall we run the unit tests before e2e in CI?

Reply:
Should be as simple as

`run: npm run test:unit`

(I'm not 100% whether we need to escape the : in test:unit)